### PR TITLE
Update e-mail-servers.rst: dedicated mailing list servers

### PR DESCRIPTION
**Pull request by ossimantylahti**
Added explanation on using dedicated servers for transactional e-mail and mass e-mails.

### CODE REVIEW

File: discuss/email_servers.rst [hunk 1]
```diff
@@ -176,3 +176,13 @@ alias in your mail server.
    
 .. _Office 365 documentation:
     https://support.office.com/en-us/article/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-office-365-69f58e99-c550-4274-ad18-c805d654b4c4
+
+Using different dedicated servers for transactional e-mail and mass e-mails
+---------------------------
+In Odoo it is possible to use two separate e-mail mail transfer agent (MTA) servers for transactional e-mails and mailing list uses. For example, it would be possible to use Odoo's own e-mail servers for transactional e-mails and Sendgrid / Amazon SES / Mailgun for mass mailings. This actually would be a good idea since Odoo's e-mail servers have a limit of sending maximum 200 e-mails per day. Alternative is to use Postmark for transactional e-mails and Amazon SES or Sendgrid for mass mailings.
```
[the remark this answers is not in the document]
Comment (kbr-odoo) [reply discuss/email_servers.rst:182]: For all contributed text: please, break lines before the 100 [character](https://www.odoo.com/documentation/user/13.0/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character).

File: discuss/email_servers.rst [hunk 2]
```diff
@@ -176,3 +176,13 @@ alias in your mail server.
...
+
+To do this, one should first enable developer mode and then enter Settings -> Technical -> Outgoing e-mail servers. There you have to create two e-mail MTA server settings. One for transactional e-mails and one for mass mail servers. Be sure to mark the priority of transactional e-mail servers as lower number as the mass email servers.
+
+Now, enter to the Settings -> Marketing by e-mail and tick box Dedicated server. This opens a selector where you can set the server for mass-mails only. With this settings Odoo will use the lower-number e-mail server for transactional emails and the server you selected for mass-mails.
+
+Note that in this case you have to set your domain's SPF records to include both transactional server and mass e-mail server. If your server resides with xxxx.odoo.com, only available options are Sendinblue and Mailchip as your e-mails would be originating from xxxx.odoo.com domain.
```
[the remark this answers is not in the document]
Comment (kbr-odoo) [reply discuss/email_servers.rst:188]: dedicated_server png

**Comment by ossimantylahti** [comment]
Suggestios done by kbr-odoo is ok and merge request has been updated accordingly.

File: discuss/email_servers.rst [hunk 3]
```diff
@@ -193,6 +193,15 @@ Set up different dedicated servers for transactional and mass mails
...
-Now, enter to the Settings -> Marketing by e-mail and tick box Dedicated server. This opens a selector where you can set the server for mass-mails only. With this settings Odoo will use the lower-number e-mail server for transactional emails and the server you selected for mass-mails.
-
-Note that in this case you have to set your domain's SPF records to include both transactional server and mass e-mail server. If your server resides with xxxx.odoo.com, only available options are Sendinblue and Mailchip as your e-mails would be originating from xxxx.odoo.com domain.
+Now, go to :menuselection:`Email Marketing --> Settings` and enable *Dedicated Server*.
+
+[image below]
+.. image:: media/IMAGE_NAME.png
+   :align: center
+   :alt: Email Marketing settings page emphasizing the option dedicated server in Odoo Email Marketing
+
```
Comment (kbr-odoo) [inline discuss/email_servers.rst:202]:
```suggestion
```

Unless the image is actually included, this can be deleted.

**Comment by kbr-odoo** [comment]
> Thank you @ossimantylahti for taking my suggestions into account :)
> I would also like to kindly ask you to have just one commit, instead of several. Afterwards, I will do a final reading.
> 
> Once again, thank you for your contribution.!

@ossimantylahti I just realized that you are not able to squash commits. Sorry about that.
So, if you could just delete that image description, or include the image, afterwards, it will be good to be merged. Thanks :)

**Comment by kbr-odoo** [comment]
Hello, @ossimantylahti :D
Shall we merge this PR? It just needs an adjustment and fixing. 👍 
Cheers!

**Comment by ossimantylahti** [comment]
> Hello, @ossimantylahti :D
> Shall we merge this PR? It just needs an adjustment and fixing. 👍
> Cheers!

I would like to merge this, but I am not sure on how to apply the fixes you requested.

**Comment by kbr-odoo** [comment]
> > Hello, @ossimantylahti :D
> > Shall we merge this PR? It just needs an adjustment and fixing. 👍
> > Cheers!
> 
> I would like to merge this, but I am not sure on how to apply the fixes you requested.

Just remove:


[image below]
.. image:: media/IMAGE_NAME.png
   :align: center
   :alt: Email Marketing settings page emphasizing the option dedicated server in Odoo Email Marketing

:) We won't be inserting the image, so this part can be removed.
Thanks!

**Comment by kbr-odoo** [comment]
@ossimantylahti Hello!
Could you proceed with the final quick adjustment requested above, so we can merge the request?
Thank you! :)

**Comment by ossimantylahti** [comment]
> @ossimantylahti Hello!
> Could you proceed with the final quick adjustment requested above, so we can merge the request?
> Thank you! :)

done

**Comment by kbr-odoo** [comment]
> > @ossimantylahti Hello!
> > Could you proceed with the final quick adjustment requested above, so we can merge the request?
> > Thank you! :)
> 
> done

I also see there is a conflict that must be resolved.
Conflicting files:
discuss/email_servers.rst

Thank you for your quick response and contribution once again :)

**Comment by ossimantylahti** [comment]
> > > @ossimantylahti Hello!
> > > Could you proceed with the final quick adjustment requested above, so we can merge the request?
> > > Thank you! :)
> > 
> > 
> > done
> 
> I also see there is a conflict that must be resolved.
> Conflicting files:
> discuss/email_servers.rst
> 
> Thank you for your quick response and contribution once again :)

I would be happy to resolve the conflicts, but I do not have permissions:

"This branch has conflicts that must be resolved
Only those with write access to this repository can merge pull requests."

**Comment by kbr-odoo** [comment]
> > > > @ossimantylahti Hello!
> > > > Could you proceed with the final quick adjustment requested above, so we can merge the request?
> > > > Thank you! :)
> > > 
> > > 
> > > done
> > 
> > 
> > I also see there is a conflict that must be resolved.
> > Conflicting files:
> > discuss/email_servers.rst
> > Thank you for your quick response and contribution once again :)
> 
> I would be happy to resolve the conflicts, but I do not have permissions:
> 
> "This branch has conflicts that must be resolved
> Only those with write access to this repository can merge pull requests."

To make it all easier, I created a PR myself with the content you suggested to be added, and I put you as the author :)
Once again, thank you for your contribution. It is appreciated!
Kindly,
Keli

**Comment by kbr-odoo** [comment]
Due to the difficulties in fixing the conflict, a new PR with the content suggested to be added was created (#881).

**Pull request closed without merging**